### DISCUSSION TITLE
Amazon Shipping Integration

### DISCUSSION THREAD

**Discussion by mumbaigirl** [AgileShift/cargo_management#6] [Ideas]
Could you integrate [Amazon Shpping API](https://developer-docs.amazon.com/amazon-shipping/docs/shipping-api-v2-reference)

**Comment by JeansReal** [comment]
Hi!
For tracking Amazon Shipments?

**Reply by mumbaigirl** [reply]
MA

> Hi! For tracking Amazon Shipments?

Amazon provides shipping services to merchants who dont sell directly on Amazon. It is called Amazon Shipping, In this case the merchnat received the order on his own website, packs the parcel and uses Amazon postal service to send it. Amazon shiiping is not associated with Amazon Marketplace. Is is a standalone shipping service provided by them

**Reply by JeansReal** [reply]
So you seems to be in need of https://github.com/frappe/erpnext-shipping
Frappe Shipping Module, that does exactly as what you describe

This Frappe app Cargo Management is not for send parcels
But rather to track parcels incoming to our warehouse
